template/src/pdk.ts.ejs

```diff
@@ -2,6 +2,30 @@
 const hostFunctions = Host.getFunctions()
 <% } %>
 
+function isNull(v: any): boolean {
+  return v === undefined || v === null
+}
+
+function cast(caster: (v: any) => any, v: any): any {
```

Comment:
cast can apply `caster` to any value. If the value is an array, then it will apply caster to every value and return a new array.